src/pip/_internal/commands/uninstall.py

```diff
@@ -51,6 +55,8 @@ def __init__(self, *args, **kw):
         self.parser.insert_option_group(0, self.cmd_opts)
 
     def run(self, options, args):
+        # type: (Values, List[Any]) -> None
+
```

Comment:
Could you please revise the method to make it return an int (always a success I think)?  BTW the exception string is a bit implicit, something like `'You must give at least one requirement to {.name} (see "pip help {.name}")'.format(self)` might be a better idea (or just hardcode the command name since `UninstallCommand` doesn't seem to be inherited anywhere).

Otherwise same as https://github.com/pypa/pip/pull/8018#issuecomment-612725333.

Reply:
Thanks @McSinyx , now all the `run` functions return an `int`. Also I haven't changed the exception string since it seems out of scope for this PR. Perhaps we can change it in another PR?